app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -415,7 +420,10 @@ describe("the new project page", () => {
     cy.get("a")
       .contains(/Quarterly reports/i)
       .click();
-    cy.get('[label*="Due Date"]').eq(0).should("have.value", "1995-01-01");
+    // the date is formatted using locale and it can show up slightly differently in different browsers (e.g., sometimes there's a period after the month abbreviation, sometimes there's not)
+    cy.get('[aria-label*="Due Date"]')
+      .eq(0)
+      .contains(/Jan[.]? 01, 1995/);
```

Comment:
I tried to enforce formatting using [`toFormat`](https://moment.github.io/luxon/api-docs/index.html#toformat) but the headless browsers still add a period to abbreviate the month. Thoughts?